src/general/StringUtils.cs

```diff
@@ -189,30 +189,29 @@ public static string Unescape(this string input)
     }
 
     /// <summary>
-    // utility function to get the correct text for the ATP label, taking
-    // a number as an input. If greater than 100, it should use F0, otherwise
-    // it should use F1. Above 1000 it should use F0 and add a K.
+    ///   Convert a value in a string with up to 3 digits.
     /// </summary>
+    /// <remarks>
+    ///   If greater than 100, it should use F0, otherwise it should use F1.
+    ///   Above 1000 it should use the number formatting which uses K for
+    ///   thousands, M for millions etc.
+    /// </remarks>
```

Comment:
Check the style of `<remarks>`. And I think we should add a remainder that this doesn't handle negative value.

Reply:
https://github.com/Revolutionary-Games/Thrive/blob/cf8b7746ef6542c021014dd07690be94e0daf2d7/src/general/StringUtils.cs#L57

Reply:
Ah I looked at one above and it didn't have <para>, I'll fix that as well. And will add that note.

Reply:
Wouldn't it be relatively easy to make this work with negative numbers as well or?